### ISSUE TITLE
[Bug] Controller Conflict

### ISSUE THREAD

**Issue by Maverynthia** [Daivuk/Daxanadu#2]
v. 3.0
When changing the controls for a controller it conflicts with what is "set in stone" for the menu inputs.

Using a game pad I change the Attack to B/A and Jump to A/B (depends on ProCOntroller/Xbox), however the controls for the menu stay the same so opening a menu and pressing B will close the menu. Using this on changing controls sets it to "= NONE =". 

Basically the confirm and close should be set to the new values of Attack/Jump for confirm and close, otherwise it will conflict when the controls are changed to be the opposite. So you can't confirm to open a menu and when using the inventory you can't confirm/or close the menu.

Maybe have a way to change built-in menu inputs

The only work around is to use a controller that isn't recognized and use and input mapper :/

**Comment by Daivuk** [comment]
Yes, Ive used standard controls for menus. A accept, B back/close.
I dont think menu controls should change with the game controls. It depends what you use for attack and jump, it might be awkward for menus.

Maybe like you said, having separate bindables for menu.

But if I understand the issue, you cannot set a certain game control because of this? Force it to = NONE= ?
